src/modules/import/domain/import-event-handler.ts

```diff
@@ -104,14 +135,22 @@ export class ImportEventHandler {
                     `System hat einen neuen Benutzer ${savedPersonWithPersonenkontext.person.referrer} (${savedPersonWithPersonenkontext.person.id}) angelegt.`,
                 );
             } else {
-                return this.logger.error(
+                this.logger.error(
                     `System hat versucht einen neuen Benutzer für ${importDataItem.vorname} ${importDataItem.nachname} anzulegen. Fehler: ${savedPersonWithPersonenkontext.message}`,
                 );
+
+                throw new ImportDomainError(
+                    `The creation of person with personenkontexte for the import transaction:${importVorgang.id} failed`,
+                    importVorgang.id,
+                );
             }
 
             if (!savedPersonWithPersonenkontext.person.newPassword) {
-                return this.logger.error(
-                    `Person with ID ${savedPersonWithPersonenkontext.person.id} has no start password!`,
+                this.logger.error(`Person with ID ${savedPersonWithPersonenkontext.person.id} has no start password!`);
+
+                throw new ImportDomainError(
+                    `The creation for a password for the person with ID ${savedPersonWithPersonenkontext.person.id} for the import transaction:${importVorgang.id} has failed`,
+                    importVorgang.id,
```

Comment:
If these errors occur, they aren't handled at all as far as I can see. They will just be logged and the import will never finish?

Reply:
Great observation, i set the status to FAILED, when the FE requests the import status it will see that the import has failed